std/internal/math/gammafunction.d

```diff
@@ -548,7 +550,11 @@ unittest {
 
 
 private {
-static if (floatTraits!(real).realFormat == RealFormat.ieeeExtended) {
+static if (floatTraits!(real).realFormat == RealFormat.ieeeQuadruple) {
+    enum real MAXLOG = 0x1.62e42fefa39ef35793c7673007e6p+13;  // log(real.max)
+    enum real MINLOG = -0x1.6546282207802c89d24d65e96274p+13; // log(real.min_normal*real.epsilon) = log(smallest denormal)
```

Comment:
@tsbockman yep I know, I tried to convert the real string, but obviously failed with low precision on my 64bit system.
Hence I wanted to point out that results with wolfram alpha differ, btw for `max` [it gives me](http://www.wolframalpha.com/input/?i=log(1.18973149535723176508575932662800702e4932)+in+base16):

```
0x2c5c.85fdf473de6af278ece600fcbdab54a087542d48a06307294043f0285fee8215026a32a8c36b4f6906fde3d9c2c481034dfb0912176b3604e2b217c387b932e90332ca820b965b56bf4b36fd61860709a2b353606c41270a00931efe794d0e24013d67dd3fdb5125d25a5d48a125dcef5f0f23479ae4fdb6bb8893c201
```